novawallet/Modules/AssetConversion/Service/HydraDx/HydraDxTokenConverter.swift

```diff
@@ -2,12 +2,12 @@ import Foundation
 import SubstrateSdk
 
 extension HydraDx {
-    struct LocalRemoteAssetId: Equatable {
+    struct LocalRemoteAssetId: Equatable, Hashable {
         let localAssetId: ChainAssetId
         let remoteAssetId: HydraDx.OmniPoolAssetId
     }
 
-    struct SwapPair {
+    struct SwapPair: Equatable, Hashable {
```

Comment:
Hashable includes Equatable

Reply:
Thanks. Will double check this in the next PR